pylint/plugins/hass_enforce_type_hints.py

```diff
@@ -146,7 +146,7 @@ class ClassTypeHintMatch:
     "mqtt_mock_entry": "MqttMockHAClientGenerator",
     "recorder_db_url": "str",
     "recorder_mock": "Recorder",
-    "requests_mock": "requests_mock.Mocker",
+    "requests_mock": "Mocker",
```

Comment:
Forcing the namespace would sometimes cause issues in nested functions, as the name of the fixture was the same as the namespace. We should therefore allow it to be without namespace.